### ISSUE TITLE
How to log in without using the mouse?

### ISSUE THREAD

**Issue by buffalotheory** [sddm/sddm#976]
How does one enter a password in the greeter login screen **without** clicking on the password input box first?

What about switching the selected user?

**Comment by adocampo** [comment]
Which theme do you use? I have breeze slightly modified because the computers here can be multi-seat, and it ask for a username and password, blinking the cursor on the password field, so last logged in user should just to **type** his/her password and press enter.

Pressing "Tab" jupms from field to filed, so "Shift+Tab" (or several Tab pressings) jump back to username and **type** it (not selecting it from a fancy list).

Take a look on the [Arch linux wiki](https://wiki.archlinux.org/index.php/SDDM)https://wiki.archlinux.org/index.php/SDDM) where they say how to create a new sddm.conf and most of the parameters you may need.

My relevant configuration looks like
`
[Autologin]
# Autologin again on session exit
Relogin=false
# Autologin session
Session=
# Autologin user
User=

[Theme]
# Current theme name
Current=breeze

[Users]
# Default $PATH
DefaultPath=/bin:/usr/bin

# Hidden shells
# Users with these shells as their default won't be listed
HideShells=
# Hidden users
HideUsers=
# Maximum user id for displayed users
MaximumUid=60000
# Minimum user id for displayed users
MinimumUid=100000
# Remember the session of the last successfully logged in user
RememberLastSession=true
# Remember the last successfully logged in user
RememberLastUser=true
`

**Comment by buffalotheory** [comment]
Thank you for your prompt feedback, and your patience in waiting for my response.

The issue is slightly-different than I understood when I posted this.  It turns out that I can log in without the mouse.

What is not working quite correctly is that the dots that appear in the password input field do not show unless this field is first clicked.  There is no cursor in the input box and no feedback for entering the password, giving the impression that the user input is not accepted.  When enter is pressed,  the dots flash briefly in the input box before the login commences.

If one clicks the password input textbox prior to entering the password, the cursor shows up in the input box and dots appear when each key is pressed, as one would expect.

I'm currently using sddm 0.17.0-r1 in the Gentoo distribution
Default configuration - no /etc/sddm.conf
The default theme seems to be 'elarun'

The configuration shown above has an extremely-high MinimumUid; even higher that the maximum.  Will this work?

I'm not sure if I should close this issue.  It works for me, but I can see this being a usability issue for others.

I'm guessing that it's more of a Qt issue than an SDDM issue...

**Comment by adocampo** [comment]
You're welcome.

I don't know if it can be related to you, but this behaviour (no dots when entering the password) somehow happens to me with sddm when I'm in a multi-monitor setup... sddm shows on both monitors but not always it detects the main monitor and when I type the password, it realy prints the dots on the secondary (or tertiary) monitor instead the main one.

It also happens on KDE lock screen, so as you, I guess is something more upstream than SDDM itself. I guess one effective way to solve it would be lock/login screen detects the main screen and show the logon form only in that screen... I think it is easy to develop and will solve some problems and why not, the unnecessary show of N logon forms.

Regarding the UIDs on sddm.conf, I have the extremely high UID because I'd set up Plasma on a 8000+ users environment, and their UID is built from samba idmap, which starts on 100000 in my scenario. Yours probably will be different and as you say, a lot lower than mine. 

I'm not sure what I set the maximun lower than the minimun, because I did it more than a year ago and sadly I didn't documented why, but I'm pretty sure that my Intention was show the username box empty.
